exercises/pangram/pangram_test.go

```diff
@@ -0,0 +1,35 @@
+package pangram
+
+import (
+	"testing"
+)
+
+const targetTestVersion = 1
+
+type testCase struct {
+	input    string
+	expected bool
+}
+
+var stringTestCases = []testCase{
+	{"quick movement of the enemy will jeopardize five gunboats", false},
+	{"the quick brown fish jumps over the lazy dog", false},
+	{"7h3 qu1ck brown fox jumps ov3r 7h3 lazy dog", false},
+	{"the 1 quick brown fox jumps over the 2 lazy dogs", true},
+	{"The quick brown fox jumps over the lazy dog", true},
```

Comment:
We have a canonical set of test data that we encourage all of the tracks to use: https://github.com/exercism/x-common/blob/master/exercises/pangram/canonical-data.json

If you see any edge cases that you've included and the canonical json is missing, please open an issue in https://github.com/exercism/x-common/issues so that we can discuss it.